documentation/modules/upgrading-mtv-ui.adoc

```diff
@@ -64,12 +64,16 @@ $ {oc} get pods -n {namespace}
-forklift-operator-6bf45b8d8-qps9v     1/1    Running  0         5m
-forklift-ui-7cdf96d8f6-xnw5n          1/1    Running  0         2m
+NAME                                                    READY   STATUS    RESTARTS   AGE
+forklift-api-bb45b8db4-cpzlg                            1/1     Running   0          6m34s
+forklift-controller-7649db6845-zd25p                    2/2     Running   0          6m38s
+forklift-must-gather-api-78fb4bcdf6-h2r4m               1/1     Running   0          6m28s
+forklift-operator-59c87cfbdc-pmkfc                      1/1     Running   0          28m
+forklift-ui-plugin-5c5564f6d6-zpd85                     1/1     Running   0          6m24s
+forklift-validation-7d84c74c6f-fj9xg                    1/1     Running   0          6m30s
+forklift-volume-populator-controller-85d5cb64b6-mrlmc   1/1     Running   0          6m36s
 ----
 +
-. If you added VMware source providers to {project-short} 2.2, upgrading to {project-short} 2.3 changes the state of any VMware providers to Critical. To fix this problem, edit the VMwhere provider by adding a VDDK `init` image and verifying the certificate of the VMware provider. For more information see link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.2/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console#adding-source-provider_vmware[Addding a VMSphere source provider].
+. If you added VMware source providers to {project-short} 2.2, upgrading to {project-short} 2.3 changes the state of any VMware providers to Critical. To fix this problem, edit the VMware provider by adding a VDDK `init` image. For more information see link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.4/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console#adding-source-provider_vmware[Addding a VMSphere source provider].
 
-. If you mapped to NFS on the {ocp} destination provider in {project-short} 2.2, upgrading to {project-short} 2.3 invalidates the NFS mapping. To fix this problem, edit the `AccessModes` and `VolumeMode` parameters in the NFS storage profile. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html-single/virtualization/index#virt-customizing-storage-profile_virt-creating-data-volumes[Customizing the storage profile].
+. If you mapped to NFS on the {ocp} destination provider in {project-short} 2.2, upgrading to {project-short} 2.3 invalidates the NFS mapping. To fix this problem, edit the `AccessModes` and `VolumeMode` parameters in the NFS storage profile. The same applies for upgrading to {project-short} 2.4. For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html-single/virtualization/index#virt-customizing-storage-profile_virt-creating-data-volumes[Customizing the storage profile].
```

Comment:
I'd go with a similar approach here as well so we won't need to include 2.5 later (at some point it would make sense to drop all things about 2.2)